minutes/2023-2024/semester-2/2023-2024-semester-2.md

```diff
@@ -1351,17 +1351,17 @@ __Late:__
             - `chair` explains Dell's markups and pricings. 
             - iFixit kit application was denied, need to explain the difference between other tools
             - Analytics server will be good, should be ready in the next day or two
-            - `events`bie says mail not broke!  (emailed `sysadmin` while he was an eepy weepy comfy guy.) We have 2 weeks to update otherwise blacklisted but lol nvm because of basic security. 
+            - `events`bie says mail not broke!  (emailed `sysadmin` while they was an eepy weepy comfy guy.) We have 2 weeks to update otherwise blacklisted but lol nvm because of basic security. 
```

Comment:
```suggestion
            - `events` says mail not broke!  (emailed `sysadmin` while they were an eepy weepy comfy guy.) We have 2 weeks to update otherwise blacklisted but lol nvm because of basic security. 
```